boards/arm/fk7b0m1_vbt6/fk7b0m1_vbt6.dts

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2023 Charles Dias <charlesdias.cd@outlook.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/h7/stm32h7b0Xb.dtsi>
+#include <st/h7/stm32h7b0vbtx-pinctrl.dtsi>
+#include <zephyr/dt-bindings/memory-attr/memory-attr-arm.h>
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+
+/ {
+	model = "FANKE FK7B0M1-VBT6 board V1.0";
+	compatible = "fanke,fk7b0m1-vbt6";
```

Comment:
This vendor prefix needs to be added to https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/bindings/vendor-prefixes.txt

Reply:
Done.